### ISSUE TITLE
How do we handle accessibility issues in issue queue?

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdrop-community#5]
I've gotten an offline request for information about Backdrop CMS handles accessibility issues. The question was do we tag accessibility issues?

We don't really use the Github labels for this purpose. We do tag issues by adding prefixes to issue titles such as:

[UX] = User Experience
[DX] = Developer Experience
[D8] = Drupal 8 Equivalent
[D7] = Drupal 7 Equivalent
[WP] = Wordpress Equivalent

[A11y] = This seems to have been used by some to tag accessibility issues. 

Are these "tags" documented anywhere?

I'm not sure how visible this method of tagging is, especially to newer community members. Is this the only way that we have to handle accessibility issues?

I believe that accessibility issues also fall under the purview of the bi-weekly design/UX meeting.

**Comment by stpaultim** [comment]
OK, I found some documentation here: https://backdropcms.org/core-issue-queue

According to this documentation, we are using [UX] for accessibility issues. 

"[UX] = User experience. This issue will make Backdrop easier to use. (All accessibility issues should be tagged [UX])."

The following issues are examples of where some users are not aware of this and/or encouraging us to use a more specific tag:

[A11Y] Dashboard: incorrect semantic structure in heading tags on the "Welcome to Backdrop" block
https://github.com/backdrop/backdrop-issues/issues/5038

[A11y] Broken skip link with flexible layout templates
https://github.com/backdrop/backdrop-issues/issues/5111
